modules/virt-creating-a-vm-from-a-template-with-an-attached-boot-source.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-creating-and-using-boot-sources.adoc
+
+[id="virt-creating-a-vm-from-a-template-with-an-attached-boot-source_{context}"]
+= Creating a virtual machine from a template with an attached boot source
+
+After you add a boot source to a template, you can create a new virtual machine from the template.
+
+.Procedure
+
+. In the {product-title} web console, click *Workloads > Virtualization* in the side menu.
+. From the *Virtual Machines* tab or the *Templates* tab, click *Create* and select *Virtual Machine with Wizard*.
+. In the *General* step, select an OS from the Operating System list that has the *(Source available)* label next to the OS and version name. The *(Source available)* label indicates that a boot source is available for this OS.
+. Ensure that the *Clone available operating system source* checkbox is selected.
+. Click *Review and Confirm*.
+. Review your virtual machine settings and edit them, if required.
+. Click *Create Virtual Machine* to create your virtual machine. The *Successfully created virtual machine* page displays with links to *See virtual machine details* or *Go to list*, to display the list of virtual machines and virtual machine templates.
```

Comment:
I would suggest rewriting the step result (The *Successfully created virtual machine* page displays....) for more clarity. It might also be better to split it into two sentences.

Reply:
I took about half of it out as these options are plainly on the UI.